about/system_requirements.rst

```diff
@@ -219,15 +219,18 @@ Desktop or laptop PC - Minimum
 |                      |                                                                                         |
 |                      |  - *Example: Intel Core 2 Duo E8200, AMD Athlon XE BE-2300, Raspberry Pi 4*             |
 +----------------------+-----------------------------------------------------------------------------------------+
-| **GPU**              | - **Forward+ rendering method:** Integrated graphics with full Vulkan 1.0 support       |
+| **GPU**              | - **Forward+ rendering method:** Integrated graphics with full Vulkan 1.0 support,      |
```

Comment:
```suggestion
| **GPU**              | - **Forward+ renderer:** Integrated graphics with full Vulkan 1.0 support,      |
```
If we're changing these already, we might want to use "renderer" here, since this is a very user-facing place (see https://github.com/godotengine/godot-docs/pull/10183, https://github.com/godotengine/godot/pull/98744 for full reasoning).

I don't already have this change in https://github.com/godotengine/godot-docs/pull/10183 because "rendering method" is still a valid term and I didn't want to make unecessary changes. But if we're already changing this page, we should use "renderer" here IMO.

(You'll have to update the RST tables to have the right number of spaces, I can't figure it out in the web UI)

Reply:
Done.